services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/RouteOptions.java

```diff
@@ -201,6 +201,25 @@ public static Builder builder() {
     return ParseUtils.parseBearings(bearings());
   }
 
+
+  /**
+   * A radius around a starting point where the routes' provider tries to avoid any
+   * significant maneuvers. Use this option when the vehicle is traveling at a significant
+   * speed to avoid dangerous maneuvers when re-routing. If a route is not found using
+   * the specified value, it will be ignored. Note that if a large radius is used, the routers'
+   * provider may ignore an important turn and return a long straight path before the first
+   * maneuver.
+   * <p>
+   * Note: the param cannot be used with {@link #departAt()} and {@link #arriveBy()},
+   * since these modes are used for reference requests, not for real-time routing.
+   *
+   * @return a radius around a starting point where router's provider tries to avoid any
+   *   significant maneuvers. Possible range is [0, 1000].
```

Comment:
```suggestion
   * @return a radius around a starting point where router's provider tries to avoid any
   *   significant maneuvers. Possible range is [0, 1000] in meters.
```